polymorphic/compat.py

```diff
@@ -0,0 +1,10 @@
+import django
+
+DJANGO_GTE_2 = django.VERSION >= (2, 0)
```

Comment:
I'd place `django.VERSION >= (2, 0)` or even better: `django.VERSION < (2, 0)` directly inline in the code

Reply:
Sure ok. WRT this and your other comment in this `compat` file, please excuse me, I was working on habit here. I've found that it's helpful to have a single point of reference for compatibility hacks, as it makes maintenance easier down the road. I probably shouldn't have introduced it here without running it past you first though :)

I'll move this inline, as you have suggested.